packages/bierzo-wallet/src/utils/test/persona.ts

```diff
@@ -47,8 +48,12 @@ export async function openEnqueuedRequest(extensionPage: Page): Promise<void> {
   await extensionPage.click("ul > li > div");
 }
 
-export async function acceptEnqueuedRequest(extensionPage: Page): Promise<void> {
-  await openEnqueuedRequest(extensionPage);
+export async function acceptEnqueuedRequest(browser: Browser): Promise<void> {
+  const extensionPage = await createExtensionPage(browser);
+  await extensionPage.bringToFront();
+
+  // click on first request
+  await extensionPage.click("ul > li > div");
```

Comment:
Maybe it is better to move common code from `acceptEnqueuedRequest` and `rejectEnqueuedRequest` to separate method?